test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTLSCertificateValidationTests.cs

```diff
@@ -29,7 +29,7 @@ public async Task DotnetRestore_withTLSCertificateValidationDisabled_DoesnotThro
         {
             // Arrange
             using var pathContext = _dotnetFixture.CreateSimpleTestPathContext();
-            TestDirectory packageSourceDirectory = TestDirectory.Create();
+            using TestDirectory packageSourceDirectory = TestDirectory.Create();
```

Comment:
You can also consider pathContext, It already creates a working directory itself, so we can just do something like Path.Combine(pathContext.WorkingDirectory, "sourceForTLSServer")